components/registration/default-registration-form.html

```diff
@@ -6,15 +6,23 @@
             <td style="font-size: 9pt">
                 F‑nr/hjelpenummer
             </td>
+            <td colspan="2">
+                <input class="form-control" ng-model="fNrOrEquivalentGetterSetter" ng-model-options=" {getterSetter: true }" maxlength="11"/>
+            </td>
+        </tr>
+        <tr>
+            <td></td>
             <td>
-                <input class="form-control" ng-model="labTestLookupNrGetterSetter" ng-model-options=" {getterSetter: true }" maxlength="11"/>
+                <button ng-click="showLabTest()" type="button" class="btn btn-secondary"  ng-disabled="!shouldShowLabTestAndVaccine()">Vis prøvesvar</button>
+
             </td>
             <td>
-                <button  ng-click="showLabTest()" class="btn btn-secondary"  ng-disabled="!shouldShowLabTest()">Vis prøvesvar</button>
+                <button ng-click="showVaccine()" type="button" class="btn btn-secondary" ng-disabled="!shouldShowLabTestAndVaccine()">Vis vaksinestatus</button>
             </td>
         </tr>
     </table>
-    <div ng-show="!shouldShowLabTest()" class="alert alert-warning" style="font-size: 9pt;">Gyldig nasjonalt felles hjelpenummer eller F-nummer må være satt for å vise prøvesvar</div>
+    <div ng-show="!shouldShowLabTestAndVaccine()" class="alert alert-warning" style="font-size: 9pt;">Gyldig nasjonalt felles hjelpenummer eller F-nummer må være satt for å vise prøvesvar og vaksinestatus</div>
+    <hr ng-if="selectedProgram.id == 'uYjxkTbwRNf' && selectedEnrollment.enrollment" />
```

Comment:
What is this line for - and why is it showing for this program? Should it rather use shouldShowLabTestAndVaccine()?

Reply:
The line is used to separate prøvesvar and vaksine from klinikermelding. (see line 26). It felt like it was needed to not get the buttons to cluttered. But open for suggestions for other ways to clean things up

![image](https://user-images.githubusercontent.com/34539750/129530859-9940d577-b6a4-4d32-b19b-3ab0d753fdae.png)

Reply:
Ah got it! It is only needed for the covid case program, as that's the only place you have both the lookups and the button for klinikermelding.